test/integration/graphql_test.lua

```diff
@@ -2147,3 +2147,225 @@ function g.test_non_finite_float()
             query_schema, nil, nil, {variables = variables})
     end
 end
+
+g.test_arguments_nullability = function ()
+    local Nullable = true
+    local NonNullable = false
+    local test_enum = types.enum({
+        name = 'My_Enum',
+        values = {
+            a = { value = 'a' },
+            b = { value = 'b' },
+        },
+    })
+
+    local graphql_types = {
```

Comment:
Should we add "object" and "list" types here?

Reply:
Done for Object
For List TBD later